lib/connect-dynamodb.js

```diff
@@ -44,6 +44,42 @@ module.exports = function (connect) {
         this.readCapacityUnits = null == options.readCapacityUnits ? 5 : parseInt(options.readCapacityUnits,10);
         this.writeCapacityUnits = null == options.writeCapacityUnits ? 5 : parseInt(options.writeCapacityUnits,10);
 
+        if (options.payPerRequest) {
```

Comment:
Why don't update the document to tell the user that if they want on-demand, set the value to z0